packages/framework/src/Resources/views/Admin/Inline/Menu/menu.html.twig

```diff
@@ -1,6 +1,29 @@
 <ul class="js-toggle-menu navig navig--side">
 
     {% if isMultidomain() %}
+        <li class="js-toggle-menu-item navig__item">
+            <div class="box-dropdown">
+                <div
+                        class="box-dropdown__select box-dropdown__select--menu"
+                        title="{{ 'Switch locale'|trans }}"
+                >
+                    <i class="svg svg-flag"></i>
```

Comment:
Personally, I found the localeFlag sufficient. This icon seems unnecessary to me

Reply:
yeah, that was just to make the button wider so the options below have some space :smile: Hopefully I will solve this by styling the proper elements as you suggested below

Reply:
fixed in 7cca405361